src/components/Announce/blogpost.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
If possible I'd like a centralized way of updating the announcement banner, rather than having blog posts be announceed via code changes and releases via PRs.

Maybe we can make PRs in the blog repo and read the API for that repo?